src/dune_rules/expander.ml

```diff
@@ -51,7 +51,7 @@ type t =
   ; bin_artifacts_host : Artifacts.Bin.t
   ; bindings : value Pform.Map.t
   ; scope : Scope.t
-  ; scope_host : Scope.t
+  ; scope_host : Scope.t option
```

Comment:
Why does the host scope have to be an option now?

Reply:
oh that's a good point. It doesn't.